src/it/mskins-28/src/site/site.xml

```diff
@@ -1,56 +1,58 @@
-    <logo name="Fourth Logo" img="./images/logos/maven-feather.png" href="http://maven.apache.org/"/>
-    <logo name="Fifth Logo" img="./images/logos/maven-feather.png" href="http://maven.apache.org/"/>
-  </poweredBy>
-  <!-- END SNIPPET: skin-custom-config -->
-
-</project>
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+ -->
+
+<project xmlns="http://maven.apache.org/DECORATION/1.1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/DECORATION/1.1.0 http://maven.apache.org/xsd/decoration-1.1.0.xsd"
+  name="${skinName}">
+
+  <skin>
+    <groupId>${skinGroupId}</groupId>
+    <artifactId>${skinArtifactId}</artifactId>
+    <version>${skinVersion}</version>
+  </skin>
+
+  <body>
+    <breadcrumbs>
+      <item name="Apache Maven Fluido Skin"  href="https://maven.apache.org/skins/maven-fluido-skin/index.html" />
+      <item name="Maven Fluido Skin ITs"  href="https://maven.apache.org/skins/maven-fluido-skin/ITs.html" />
+    </breadcrumbs>
+    <menu name="Overview">
+      <item name="Introduction" href="index.html"/>
+    </menu>
+  </body>
+
+  <!-- START SNIPPET: skin-custom-config -->
+  <poweredBy>
+    <logo name="First Logo" alt="First Logo" img="./images/logos/maven-feather.png" href="http://maven.apache.org/"/>
+    <!-- Second logo is wider than the sidebar -->
+    <logo name="Second Logo" alt="Second Logo" width="500px" img="./images/logos/maven-feather.png" href="http://maven.apache.org/"/>
+    <!-- Third and forth logo fit next to eachother in the sidebar -->
+    <logo name="Third Logo" alt="Third Logo" img="./images/logos/maven-feather.png" href="http://maven.apache.org/"/>
+    <logo name="Fourth Logo" alt="Fourth Logo" img="./images/logos/maven-feather.png" href="http://maven.apache.org/"/>
+    <logo name="Fifth Logo" alt="Fifth Logo" img="./images/logos/maven-feather.png" href="http://maven.apache.org/"/>
+    <logo name="Sixth Logo" alt="Sixth Logo" img="./images/logos/maven-feather.png"/>
+    <logo name="Seventh Logo"/>
+  </poweredBy>
+  <!-- END SNIPPET: skin-custom-config -->
+
+</project>
```

Comment:
why don't we see the real diff? is there a CR/LF change?
I see 2 options:
- the CRLF change is accidental: let's avoid it
- the CRLF change is a fix to previous problem: let's do the fix in a separate commit

Reply:
This happens regularly because many files have been created before .gitattributes and never touched since. This is the outcome. So the change was not on purpose.

Reply:
This affects only this single file, do you still want me to try to "fix" previos LS?

Reply:
@hboutemy Done, have another look.